test/test_configuration.rb

```diff
@@ -1,23 +1,25 @@
-require 'helper'
+# rubocop:disable Metrics/ClassLength
```

Comment:
The only way to do that would be to exclude all of tests from the metrics (I'm not against this, I wonder what @parkr thinks?)  Other than that there is no way other than by file.